app/models/job/Worker.scala

```diff
@@ -81,6 +83,12 @@ class WorkerService @Inject()(conf: WkConf) {
       "lastHeartBeatIsRecent" -> lastHeartBeatIsRecent(worker)
     )
 
+  def assertDataStoreHasWorkers(dataStoreName: String)(implicit ctx: DBAccessContext): Fox[Boolean] =
+    for {
+      _ <- dataStoreDAO.findOneByName(dataStoreName)
+      _ <- workerDAO.findOneByDataStore(dataStoreName)
+    } yield true
```

Comment:
```suggestion
    } yield ()
```
(since the boolean return value is never read, I’d instead return unit)